src/pages/components/Background.qml

```diff
@@ -38,13 +38,13 @@ Item {
 
         Rectangle {
             color: Theme.overlayBackgroundColor ? Theme.overlayBackgroundColor : "black"
-            opacity: 0.948
+            opacity: 0.948 // TODO: constants?
             anchors.fill: parent
         }
 
         Image {
             id: glassTextureImage
-            opacity: 0.1
+            opacity: 0.1 // TODO: constants?
```

Comment:
By heart afair these are for background texture to work properly. I'd just remove that "TODO: contants".